src/Components/CreateImageWizardV2/steps/FileSystem/FileSystemConfiguration.tsx

```diff
@@ -63,6 +119,17 @@ const FileSystemConfiguration = () => {
       {partitions?.find((partition) =>
         partition?.mountpoint?.includes('/usr')
       ) && <UsrSubDirectoriesDisabled />}
+      {!isNextButtonPristine &&
+        getDuplicateMountPoints(partitions)?.length !== 0 &&
+        getDuplicateMountPoints(partitions)?.length !== undefined && (
+          <div style={{ padding: '15px 0' }}>
```

Comment:
The spacing looks a bit weird here. I think we can go with default one and remove the "style"

```suggestion
          <div>
```